samples/net/wifi/sample.yaml

```diff
@@ -53,6 +53,7 @@ tests:
       - nrf7002dk/nrf5340/cpuapp/nrf7001
   sample.net.wifi.nrf7002ek:
     extra_args:
+      - SB_CONFIG_WIFI_NRF70=y
```

Comment:
@PerMac can this instead be reworked to use the alt config root?

Reply:
Sure, I need to checkout this `alt` feature, I will take it up in a separate PR, thanks.